.github/workflows/testing.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       serverlessIndexName: ${{ steps.step3.outputs.SERVERLESS_INDEX_NAME }}
+      assistantName: ${{ steps.step3.outputs.ASSISTANT_NAME }}
+      testFile: ${{ steps.step3.outputs.TEST_FILE }}
```

Comment:
@Reviewers: You can ignore these diffs if you like. It's just me allowing CI to spin up a global Assistant and a file to play with.